### PR TITLE
feat(wallet): phantom integration on the Iridium-dev

### PR DESCRIPTION
**Pull request by JustZacca**
As the name implies, this PR introduces Phantom Wallet to the Satellite, allowing users to sign up and run Iridium with it.
This PR also modifies the classic initialization of Iridium.

Now both by connecting the external wallet and using the internal wallet, iridium starts up by signing a message.

I corrected the tests that I could correct, those related to my PR, but according to Manuel's directives, I completely disabled the non-working tests.

This PR is temporary, as much as it does its job, it is only a small part, as we still have to modify the store and part of the workflow.

### CODE REVIEW

File: store/accounts/actions.ts [hunk 1]
```diff
@@ -255,8 +269,8 @@ export default {
       photoHash: imagePath,
       address: walletAccount.publicKey.toBase58(),
     })
-    dispatch('initializeEncryptionEngine', walletAccount)
-    dispatch('startup', walletAccount)
+    // dispatch('initializeEncryptionEngine', walletAccount)
```
Comment (stavares843) [inline store/accounts/actions.ts:272]: do we need this commented out? 🔨

File: libraries/Phantom/PhantomFriends.ts [hunk 2]
```diff
@@ -0,0 +1,678 @@
...
+      this.buildEventHandler(FriendsEvents.NEW_REQUEST),
+      Config.solana.defaultCommitment,
+      [incomingRequestsFilter],
+    )
+
+    // Filter for new friends checks only if an outgoing request has been accepted
+    // because we suppose the incoming request acceptance to be catch directly after the
+    // success of the acceptRequest action
+    // This filter checks the sender public key (our) and the status
+    // [32 bytes (sender public key)][1 byte (status)][32 bytes (recipient public key)]
```
Comment (stavares843) [inline libraries/Phantom/PhantomFriends.ts:576]: maybe we could use multi-line comment instead for this

**Review by stavares843 — COMMENTED** [review]
can you skip the tests using

image


instead of renaming pls? thanks 🔨

**Comment by iltumio** [comment]
> > can you skip the tests using
> > image
> > instead of renaming pls? thanks 🔨
> 
> I changed the name of the files according to Manuela's instructions, but if you want I'll use skip instead

the rename is better because if we have errors like missing imports due to a whole library change it will fail even if you skip it

**Comment by stavares843** [comment]
yes i know those was the instructions before 🔨 , but in case there are no imports missing or anything is better to just skip the test and just renaming only if really needed

**Comment by stavares843** [comment]
this PR is re-adding things that are already on iridium-dev so some rebase was made incorrectly

**Pull request closed without merging**